target-design/switch/switch.cc

```diff
@@ -11,7 +11,7 @@
 #include <omp.h>
 #include <cstdlib>
 
-#define IGNORE_PRINTF
+//#define IGNORE_PRINTF
```

Comment:
why is this commented out?

Reply:
Whoops. I thought I reverted those changes, but apparently not.